module/Motion/PwmRotation.h

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file   Rotation.h
```

Comment:
```suggestion
 * @file   PwmRotation.h
```
コメントですが。.cppの方と合わせるとこう。